andela/banasko-cfh
# #134249129 login with jwt

#### What does this PR do?             
This PR ensures that once a user successfully login,  they receive a JSON web token for further authentication.
#### Description of Task to be completed?   
When  a user successfully login, a json web token is sent back to that user. The token sent back is used to access restricted routes. 
#### How should this be manually tested?     
This PR can be tested manually by running a local server and using POSTMAN. User should send their valid details (username and password) to this endpoint "localhost:port/api/auth/login". The port should be set in the .env file.
